src/vs/workbench/contrib/webview/browser/pre/index.html

```diff
@@ -791,6 +791,17 @@
 			}
 		}
 
+
```

Comment:
this listens for any drag events that include data. It does this even if the extension is also handling the drag event. 

**note** VSCode File Drag and drop events do not have dataTransfer.items